spec/lib/os_places_api/client_spec.rb

```diff
@@ -23,6 +23,8 @@
             "RPC" => "2",
             "X_COORDINATE" => 533_813.0,
```

Comment:
So `X_COORDINATE` and `Y_COORDINATE` still get returned when `output_srs` is WGS84?

Reply:
Yes, the only difference is that we get the two new fields but the rest is all the same.